src/build/stable/commands/install_global_dependencies/install_dfx.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Thoughts on making all of these install_thing.sh scripts into ts files? I see pros and cons both ways

Reply:
we're keeping it sh files